package-lock.json

```diff
@@ -1,9956 +1,8 @@
 {
   "name": "pwa-kit",
   "version": "2.7.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
@vcua-mobify The `develop` branch supports Node 16 & NPM7/8 and Node 14 & NPM 6.
The `package-lock.json` files were updated in [this PR](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1012) to use `lockfileVersion: 2` which is backwards compatible with `lockfileVersion: 1` to fix the warning users see if we use `lockfileVersion:1` with Node 16 with NPM 7 or NPM 8.
https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json#:~:text=Backwards%20compatible%20to%20v1%20lockfiles

We want to keep supporting Node 16 and Node 14 on `develop`. Please, regenerate lock files using NPM8 to use `lockfileVersion: 2`.